libs/openFrameworks/events/ofEvents.h

```diff
@@ -213,6 +213,8 @@ class ofCoreEvents {
 	ofEvent<ofMouseEventArgs> 	mousePressed;
 	ofEvent<ofMouseEventArgs> 	mouseReleased;
 	ofEvent<ofMouseEventArgs> 	mouseScrolled;
+	ofEvent<void>				mouseEntered;
+	ofEvent<void>				mouseExited;
```

Comment:
isn't there any way to know where the mouse entered? when it exists we can for sure use the last position and when it enters we can probably use it too if there's a mouse position triggered before mouse entered?  it would be useful to report that too even if GLFW doesn't do it by default